_posts/2020-09-15-open-sourcing-a-private-project.markdown

```diff
@@ -0,0 +1,89 @@
+---
+layout: post
+title: "How to open source a private project"
+date: 2020-09-17 11:00:00
+categories: ["open source"]
+author: "fionaDL"
+---
+
+Open Source is one of the things that we value as a company. Our philosophy is that “everything we do should be open by default”. This is why in the past few months we decided to open source two of our internal projects. We built [Dash](https://github.com/fastruby/dash) and [Points](https://github.com/fastruby/points) to make our daily processes smoother, and then we thought “hey, why not share them with everyone?”.
+Now we also want to share the process of how we turned them from private to open source.
+
+<!--more-->
+
+## Making a list
+
+Before we began the process of open sourcing we decided to make a list. We thought about what it would take to move the projects into the eyes of the public. We thought about what we wanted the app to look like, if there were any “secrets” in the code, and how we would present the projects. These are some of the things we came up with for our checklist.
+
+### Hiding sensitive information
+
+It is important to go through commit history and the code to make sure there are no “secrets” you would be sharing with the public when you open source. We ended up needing to squash the commit history because we had some areas that had sensitive information. Maybe your code is already free of sensitive information, but it is a good idea to check.
+
+Look at places where you may have hardcoded company specific or personal information. We often move these hardcoded pieces of code into environment variables. Just make sure you include the environment variables and an explanation in the README.
```

Comment:
This sounds a bit confusing: Just make sure you include the environment variables and an explanation in the README.

I think you mean something like this: Just make sure you include a sample file with the environment variables (and sample values) and an explanation in the README. Something like `.env.sample` so future users and contributors have an idea of what those look like.